src/js/view/utils/views-manager.js

```diff
@@ -56,11 +62,19 @@ document.addEventListener('webkitvisibilitychange', onVisibilityChange);
 window.addEventListener('resize', scheduleResponsiveRedraw);
 window.addEventListener('orientationchange', scheduleResponsiveRedraw);
 
+if (screen.orientation) {
+    screen.orientation.addEventListener('change', checkOrientation);
```

Comment:
Don't add event listeners for code that is user-agent guarded. If this functionality is only for Android Chrome, then the `OS.android && Browser.chrome` check should happen here before adding the listener, instead of making all browsers with `screen.orientation` do the same work for nothing.